### ISSUE TITLE
Change "Asset_Cost" from text to number field

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#9943]
> What application are you using?

ArcGIS

> Describe the problem.

The sign_specs_actuals in the Signs_Work_Orders_AGOL Feature Service layer has a field called "Asset_Cost" as a text field. Please change it to a Number field (double).

> How soon do you need this?

Flexible  — An extended timeline is OK

> Requested By
Susanne G.

Request ID: DTS22-104830

**Comment by andrewshensky** [comment]
@dianamartin I am thinking I could just remove this field and recreate it with the same name but as an integer, that way the script pulling the data from Knack should not be impacted. Will changing this field type impact anything with the data coming from knack?

**Comment by dianamartin** [comment]
@andrewshensky I think script like "short text" field types so easily move data from one place to another. I'll have to ask John if he thinks there is an impact. 

@johnclary Can you weigh in on if Andrew's proposed solution will cause an impact? (deleting a field, recreating it as a number field. It's for Asset Cost.

**Comment by johnclary** [comment]
@andrewshensky's approach should work! this is a "currency" field in knack, which gave me pause b/c i thought those types were stored as strings (with the dollar sign and commas), but glancing at the AGOL I believe I am mistaken. 

Let's try it! @andrewshensky however you want to do it, but if you delete and re-add the column right from AGOL we can refresh the data.

**Comment by dianamartin** [comment]
@andrewshensky Let me know when you've done this, I have an issue created for refreshing the script for the dev group when you're done recreating the field.

**Comment by andrewshensky** [comment]
@dianamartin I have it on my list of items for tomorrow (8/11). I'll post here when it has been completed.

**Comment by andrewshensky** [comment]
@dianamartin I have now deleted the existing text field and added a new field with the same name "Asset_Cost". This new field is a double, which should allow for capturing the decimal values of the asset cost. Let me know if any other changes are needed.

Before
![image](https://user-images.githubusercontent.com/80716478/184191481-de452282-46e2-41eb-aabc-5dc01c60e178.png)

After
![image](https://user-images.githubusercontent.com/80716478/184191504-c39eef6b-0423-4ae6-8296-f39f0036947c.png)

**Comment by dianamartin** [comment]
@johnclary should we create a new issue for the refresh request for this?

**Comment by johnclary** [comment]
thanks y'all! i see the asset cost is already starting to be populated correctly for new records. i'm running the refresh now. will keep you posted.

**Comment by johnclary** [comment]
@dianamartin the refresh is complete. would you ask SMB to confirm everything looks good on their side?

**Comment by dianamartin** [comment]
Messaged Susanne 8/12
>We just refreshed the "Asset_Cost" you asked for.
Can you check the field on your side?

**Comment by dianamartin** [comment]
@andrewshensky I'm still waiting on SMB to reply about assessing the new field..
